### PR TITLE
Centre collapsed buttons RSC-503

### PR DESCRIPTION
**Pull request by glenhunter**
https://issues.sonatype.org/browse/RSC-603

For the main nav buttons I added a single pixel of padding to the left and that centres the icons in the collapsed state.

The support icon had right margin on the icon that I had to remove when the sidebar is collapsed.

### CODE REVIEW

**Comment by rpokorny** [comment]
So the decision was that the icons should be 1px farther right both when it's collapsed and when it's expanded?

File: lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss [hunk 1]
```diff
@@ -92,7 +92,10 @@ $nx-global-sidebar-separator-color: var(--nx-swatch-indigo-40);
 
   border: 1px solid transparent;
   display: block;
-  padding: var(--nx-spacing-base) $nx-global-sidebar-left-right-spacing;
+  padding: var(--nx-spacing-base)
+           $nx-global-sidebar-left-right-spacing
+           var(--nx-spacing-base)
+           calc(#{$nx-global-sidebar-left-right-spacing} + 1px);
```
Comment (rpokorny) [inline lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss:98]: Could use a code comment

**Review by rpokorny — APPROVED** [review]

**Comment by glenhunter** [comment]
> So the decision was that the icons should be 1px farther right both when it's collapsed and when it's expanded?

I think that it's important that the distance be the same open and closed, the 1px jump was noticeable to me when I tried it.

File: lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss (hunk 1 above)
Comment (glenhunter) [reply lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss:100]: > Could use a code comment

https://github.com/sonatype/sonatype-react-shared-components/pull/424/commits/570d8aef9a7c6c6c8ebd0c4f104c24668002aeb5

**Pull request merged**